README.md

```diff
@@ -34,9 +34,26 @@ levels.
 ## Power Spectrum Analysis
 
 Now that sources have been masked, we can begin a power spectrum analysis
-of the dusty data.
-
-**Under Development**
+of the dusty data. A 2D FFT is performed of the data and the masked data.
+The 2D psd obtained is now azimuthally averaged to create a 1D psd, which
+represents the radial power spectrum of the masked data.
+
+## Fake Images
+
+A set of simulated is generated. The fake image has gaussian sources, whose
+parameters can be adjusted based on the following arguments. The amplitude(flux)
+of the souces are distribued according to a powerlaw, with exponent gamma. The
+default value of gamma is -1.25, in accordance with the schechter luminosity
+function. The standard deviations of the souces follow a 1/r^2 distribution.
+
+A background with spatial fluctuations at various scales is created seperately.
+This is achived by first creating the desired 2D power spectrum, which is a
+radial power law in Fourier space. According to litrature, the index of ISM
+power law distribution is -2.9. Taking the inverse FFT of this p_law array gives
+a background with the desired levels of spatial fluctuations in real space.
+
+This background is added to the set of galaxies, to create an accurate simulated
+data set
 
```

Comment:
Can you provide instructions on how to tests each of these codes? I've been doing:
```
ln -s /Users/lspitler/Documents/huntsman-dust/huntsman_dust
python huntsman_dust/power_spectrum.py
```

Now that the code is public, you might want to choose a LICENSE.md file to use. Github provides templates.
https://github.com/lspitler/astrometry.net-lite/blob/master/LICENSE.md

Reply:
This is really cool. I had no idea what symbolic links were, but they are definitely very useful!
I will update the readme.md to provide instuctions on how to run the code.

Reply:
Done. The readme now shows how to install and use the package